include/seqan/blast/blast_statistics.h

```diff
@@ -555,6 +573,35 @@ _selectSet(BlastScoringScheme<TScore> & scheme)
     return false;
 }
 
+template <typename TValue, typename TSpec>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, TSpec>>> & scheme)
+{
+    using TScore = Score<TValue, ScoreMatrix<AminoAcid, TSpec>>;
+    using TKAValues = KarlinAltschulValues<TScore>;
+    return _selectSet<TScore, TKAValues>(scheme);
+}
+
+template <typename TValue>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, ScoreSpecSelectable>>> & scheme)
```

Comment:
I know that with C++11 the spaces between closing angle brackets are optional, but yet our coding guidelines suggests the whitespaces in-between. I also do not prefer the one or the other scheme, but it would be better to make it consistent with the current code. We can adapt that for seqan3.